### DISCUSSION TITLE
how can i get auto-healing progress

### DISCUSSION THREAD

**Discussion by lddlww** [minio/minio#14257] [Q&A] [answered]
when one drive is corrupt，then auto-healing process start  healing，question is how can i check it progress？

**Comment by harshavardhana** [comment, marked as answer]
mc admin heal

**Comment by lddlww** [comment]
AFAIK，```mc admin heal``` is foreground，when auto-healing process in background，how can i check it status info？

**Reply by harshavardhana** [reply]
> AFAIK，`mc admin heal` is foreground，when auto-healing process in background，how can i check it status info？

Your understanding is wrong check `mc admin heal alias/`

**Comment by lddlww** [comment]
oh，thanks for remind
